hack/make/.integration-daemon-start

```diff
@@ -16,7 +16,8 @@ export DOCKER_GRAPHDRIVER=${DOCKER_GRAPHDRIVER:-vfs}
 export DOCKER_EXECDRIVER=${DOCKER_EXECDRIVER:-native}
 
 if [ -z "$DOCKER_TEST_HOST" ]; then
-	export DOCKER_HOST="unix://$(cd "$DEST" && pwd)/docker.sock" # "pwd" tricks to make sure $DEST is an absolute path, not a relative one
+	mkdir -p "/tmp/$DEST"
```

Comment:
Can we use `mktemp -d` here instead (and clean up after ourselves, of course)?  These can run on the host too, where this isn't as safe.